mobile/README.md

```diff
@@ -1,16 +1,62 @@
-# Polydodo
+# Polydodo mobile client
 
-A new Flutter project.
+<p align="center">
+   <a href="https://github.com/PolyCortex/polydodo/releases/latest/download/polydodo_app_android.apk">
+    <img src="https://img.shields.io/badge/Get latest release-Android-3EDDC84?logo=Android"
+         alt="web client">
+  </a>
+     <a>
+    <img src="https://img.shields.io/badge/Future release-iOS-purple?logo=Apple"
+         alt="web client">
+  </a>
+  <a href="https://github.com/dart-lang">
+    <img src="https://img.shields.io/badge/Language-Dart-blue?logo=Dart"
+         alt="web client">
+  </a>
+    <a href="https://flutter.dev/">
+    <img src="https://img.shields.io/badge/Framework-Flutter-44D1fd?logo=Flutter"
+         alt="web client">
+  </a>
+ </p>
+
+This mobile app receives, scores, saves and aggregates sleep sequences. Unlike the web application, this app can save sleep sequences for later consultation and display the aggregated results of several nights of sleep on a dashboard. Also, it will guide the user from the installation of the electrodes, until the end of his acquisition. It does not require the use of OpenBCI GUI as it directly interface itself with the OpenBCI board.
```

Comment:
```suggestion
This mobile app receives, scores, saves and aggregates sleep sequences. Unlike the web application, this app can save sleep sequences for later consultation and display the aggregated results of several nights of sleep on a dashboard. Also, it will guide the user from the installation of the electrodes, until the end of their data acquisition. It does not require the use of OpenBCI GUI since it directly interfaces with the OpenBCI board.
```